org.lflang.tests/src/org/lflang/tests/compiler/RoundTripTests.java

```diff
@@ -46,31 +46,30 @@ public void roundTripTest() throws Exception {
 
     private void run(Path file) throws Exception {
         Model originalModel = parse(file);
-        System.out.printf("Running formatter on %s...%n", file);
         Assertions.assertTrue(originalModel.eResource().getErrors().isEmpty());
         // TODO: Check that the output is a fixed point
         final int smallLineLength = 20;
-        String squishedTestCase = FormattingUtils.render(originalModel, smallLineLength);
-        System.out.printf(
-            "The following is what %s looks like after applying formatting with the preferred line "
-                + "length set to %d columns:%n%s%n%n",
-            file.getFileName(),
-            smallLineLength,
-            squishedTestCase
-        );
-        Model resultingModel = getResultingModel(file, squishedTestCase);
+        final String squishedTestCase = FormattingUtils.render(originalModel, smallLineLength);
+        final Model resultingModel = getResultingModel(file, squishedTestCase);
         Assertions.assertNotNull(resultingModel);
         if (!resultingModel.eResource().getErrors().isEmpty()) {
             resultingModel.eResource().getErrors().forEach(System.err::println);
             Assertions.assertTrue(resultingModel.eResource().getErrors().isEmpty());
         }
-        Assertions.assertTrue(new IsEqual(originalModel).doSwitch(resultingModel));
-        String normalTestCase = FormattingUtils.render(originalModel);
-        System.out.printf(
-            "The following is what %s looks like after applying formatting normally:%n%s%n%n",
-            file.getFileName(),
-            normalTestCase
-        );
+        if (!new IsEqual(originalModel).doSwitch(resultingModel)) {
+            System.out.printf(
+                "The following is what %s looks like after applying formatting with the preferred line "
+                    + "length set to %d columns:%n%s%n%n",
+                file.getFileName(),
+                smallLineLength,
+                squishedTestCase
+            );
+            throw new junit.framework.AssertionFailedError(String.format(
+                "The reformatted version of %s was not equivalent to the original file.",
+                file.getFileName()
```

Comment:
I noticed that the error reporting tests do not report the path to the offending file, like this line does. We should add it there, too.

Reply:
You are right. Done!